web/src/beta/components/Resizable/hooks.ts

```diff
@@ -142,9 +196,28 @@ export default (
   );
 
   const handleResetSize = useCallback(() => {
+    if (!localStorageKey) return;
```

Comment:
Same as my previous comment from last week. I think we should allow basic functionality even if there is issues with localstorage